schemas/create-labware/1-create-labware.avsc

```diff
@@ -91,6 +91,12 @@
                     "name": "sampleCollectionDateUtc",
                     "doc": "Sample Collection of the sample in UTC format.",
                     "type": { "type": "long", "logicalType": "timestamp-millis" }
+                  },
```

Comment:
Is it worth adding some documentation? There are a few items that do not have a detailed explanation.